src/js/components/FragmentList/fragment.jsx

```diff
@@ -79,21 +36,21 @@ const Fragment = ({
   }
 
   return (
-    <StyledFragment
+    <Styled.Fragment
```

Comment:
How about removing that Styled prefix and not differentiate between styled component and regular react components?